packages/css/src/components/tabs/tabs.template.ts

```diff
@@ -0,0 +1,39 @@
+import { Tabs } from '@dso-toolkit/sources';
+
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+export function tabsTemplate({ tabs }: Tabs) {
+  return html`
+    <ul class="nav nav-tabs" role="tablist">
+      ${tabs.map((tabitem) => {
+        return html`
+          <li
+            role=presentation
+            class=${ifDefined(tabitem.modifiers ? tabitem.modifiers : undefined)}
+            aria-selected=${ifDefined(tabitem.modifiers==='active' ? true : false)}
+            id=${tabitem.id}
+            aria-controls=${tabitem.id}-tab
+            >
+            <a href="#" role="tab">
+              ${tabitem.label}
+            </a>
+          </li>
+        `;
+      })}
+    </ul>
+    ${tabs.map((tabitem) => {
+      return html`
+        <div
+          tabindex=${ifDefined(tabitem.modifiers==='active' ? 1 : 0)}
+          role=tabpanel
+          id=${tabitem.id}-tab
```

Comment:
Ik wist niet dat dit werkte! Maar ik zie graag dat values van attributen tussen quotes staan, tenzij het _alleen_ een interpolatie is.

```suggestion
          id="${tabitem.id}-tab"
```

Reply:
Volgens mij kwam ik het in andere ook tegen zonder quotes.